Userland/Libraries/LibWeb/DOM/Document.h

```diff
@@ -411,6 +421,7 @@ class Document
     NonnullRefPtrVector<HTML::HTMLScriptElement> m_scripts_to_execute_as_soon_as_possible;
 
     QuirksMode m_quirks_mode { QuirksMode::No };
+    Type m_type { Type::HTML };
```

Comment:
```suggestion

    // https://dom.spec.whatwg.org/#concept-document-type
    Type m_type { Type::HTML };

```

...instead of next to the enum class, to match the various ones that already exist on the members below :)